### ISSUE TITLE
Accessible mathjax

### ISSUE THREAD

**Issue by tonyfast** [Iota-School/notebooks-for-all#81]
currently, mathjax equations are ignored from automated tests #80 . there should be ways to tune mathjax to improve accessibility.

https://docs.mathjax.org/en/latest/basic/accessibility.html
